trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -5,10 +5,8 @@ const userParamValidation = param('user').isMongoId().withMessage('The User ID (
 const facilityIdValidation = param('facilityId').isMongoId().withMessage('The Facility ID (facilityId) provided should be a Mongo ID');
 const amendmentIdValidation = param('amendmentId').isMongoId().withMessage('The Amendment ID (amendmentId) provided should be a Mongo ID');
 const dealIdValidation = param('dealId').isMongoId().withMessage('The Deal ID (dealId) provided should be a Mongo ID');
-const partyURNValidation = param('urn')
-  .isString()
-  .matches(/^\d+$/)
-  .withMessage('The party URN (urn) provided should be a string of numbers');
+const groupIdValidation = param('groupId').isInt().withMessage('The Group ID (groupId) provided should be an integer');
```

Comment:
This is some very basic validation added in here -- Looking at the code, I can tell that groupId always has to be an int.